packages/doenetml-iframe/src/index.tsx

```diff
@@ -259,7 +259,7 @@ export function DoenetViewer({
                     boxSizing: "border-box",
                     overflow: "hidden",
                     border: "none",
-                    minHeight: 200,
+                    minHeight: 50,
```

Comment:
Ah, I think the min-height was to accommodate the virtual keyboard at some point.